optimus/io/jdbc.py

```diff
@@ -278,20 +299,20 @@ def df_to_table(self, df, table, mode="overwrite"):
         conf.save()
 
     @staticmethod
-    def _limit(limit=None):
+    def _limit(df: DataFrame, limit=None) -> DataFrame:
```

Comment:
Type annotations are a really good idea, Can you make a separate PR for the complete codebase? I am assuming the IDEs/other tools can do this automatically for about 75% of the codebase accurately. Let me know your thoughts

Reply:
Yup kill fix that :)